weaviate/collections/queries/hybrid/generate.py

```diff
@@ -24,7 +24,7 @@ def hybrid(
         single_prompt: Optional[str] = None,
         grouped_task: Optional[str] = None,
         grouped_properties: Optional[List[str]] = None,
-        alpha: NUMBER = 0.5,
+        alpha: NUMBER = 0.7,
```

Comment:
why?

Reply:
I recall @donomii telling me that this is the number used as the server-side default so we should align it here

Reply:
I think the trouble was that the client can't send none for a number, so it was setting the value to zero?

In that case, matching the server default would be best

Reply:
Ah yes that was it, thanks for the context 😁